src/test/java/uk/gov/pay/adminusers/client/ledger/service/LedgerServiceTest.java

```diff
@@ -76,4 +78,30 @@ public void getTransaction_shouldDeserialiseLedgerPaymentTransactionCorrectly()
         assertThat(transaction.getTransactionId(), is(externalId));
         assertThat(transaction.getReference(), is("test event ref"));
     }
+
+    @Test
+    public void searchTransactions_shouldDeserialiseSearchTransactionsResponseCorrectly() throws JsonProcessingException {
+        String externalId = "e8eq11mi2ndmauvb51qsg8hccn";
+        ImmutableMap<String, Object> transactionData = ImmutableMap.of(
```

Comment:
We can use the standard java.util.Map here as it is immutable.

Reply:
Updated!